spec/tests/helpers/perf_run_helper_spec.rb

```diff
@@ -773,6 +779,26 @@ class PerfRunHelperClass
         ).to eq(true)
       end
     end
+    context "when 'orchestration_service memory delta' < MAX_BASELINE_VARIANCE_ORCH_REL_MEM AND worse" do
+      it "returns false" do
+        expect(
+          subject.validate_baseline_data(
+            "pass_thing1"                                    => [1.00, 0.98],
```

Comment:
I can't find MAX_BASELINE_VARIANCE_ORCH_REL_MEM defined anywhere...
I think it changed to MAX_VARIANCE_OVERRIDE_PROCESS_ORCHESTRATION_SERVICES_RELEASE_AVG_MEM
And the hardcoded numbers should probably be calculated based on it so that if it changes the tests don't have to change too.

Reply:
:+1: Spec tests updated to use the modules constants to calculate the variances. Fixed the incorrect reference.